packages/wonder-blocks-core/src/hooks/__tests__/use-latest-ref.typestest.ts

```diff
@@ -1,18 +1,21 @@
+import * as React from "react";
 import {useLatestRef} from "../use-latest-ref";
 
-// The correct, non-nullable type should be inferred for `current`.
-useLatestRef(123) satisfies {current: number};
+const _ = () => {
```

Comment:
**praise:** Nice way to avoid the linter; though I think I would've just disabled it for the whole file.